src/scancode/cli.py

```diff
@@ -185,6 +186,11 @@ def validate_formats(ctx, param, value):
         raise click.BadParameter('Invalid template file: "%(value)s" does not exists or is not readable.' % locals())
     return value
 
+def ignore_files(ctx, param, value):
+    for file in value.split(','):
+        user_ignored.update({
```

Comment:
@pombredanne I've [pushed all the changes](https://github.com/yashdsaraf/scancode-toolkit/commit/6401c5b406be5e6bcfb8b2c3246eae14d3782bc7) you suggested, there's just one issue that `fnmatch` does not support double wildcard, is it alright if I add a third party library [glob2](https://github.com/miracle2k/python-glob2) to enable the double wildcard support?